hudi-client/hudi-client-common/src/main/java/org/apache/hudi/client/AbstractHoodieWriteClient.java

```diff
@@ -859,6 +931,10 @@ protected void setWriteSchemaForDeletes(HoodieTableMetaClient metaClient) {
     }
   }
 
+  private boolean isEager(HoodieFailedWritesCleaningPolicy cleaningPolicy) {
```

Comment:
Like I mentioned above, we can make turn this into `eagerCleanFailedWrites()` returning boolean, working off the member variable `config` ? In its current form, this should probably reside in `HoodieFailedWritesCleaningPolicy` itself

Reply:
Responded above